appveyor.yml

```diff
@@ -20,18 +19,9 @@ environment:
     secure: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
 
   matrix:
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "27"
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "34"
-    - TARGET_ARCH: "x86"
-      CONDA_PY: "35"
-    - TARGET_ARCH: "x64"
-      CONDA_PY: "27"
-    - TARGET_ARCH: "x64"
-      CONDA_PY: "34"
-    - TARGET_ARCH: "x64"
-      CONDA_PY: "35"
```

Comment:
Actually we can merge this as it has removed the Python environment selection.

cc @pelson